src/System.Management.Automation/minishell/api/RunspaceConfigurationTypeException.cs

```diff
@@ -20,12 +20,7 @@ namespace System.Management.Automation.Runspaces
     ///     2. ISerializable
     /// -->
     [Serializable]
-#if CORECLR
-    internal
-#else
-    public
-#endif
-    class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
+    internal class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
```

Comment:
You don't need to get all such changes in a single PR. When you submitting PRs, you can identify which of them are similar changes and group them together.